File: datacrm_app/lib/domain/models/login_model.dart
class LoginResponse {
  final Result? result;
  final bool success;

  LoginResponse({required this.result, required this.success});

  factory LoginResponse.fromJson(Map<String, dynamic> json) {
    return LoginResponse(
      result: json['result'] != null ? Result.fromJson(json['result']) : null,
      success: json['success'],
    );
  }
}

class Result {
  final String sessionName;
  final String userId;
  final String version;
  final String vtigerVersion;

  Result({
    required this.sessionName,
    required this.userId,
    required this.version,
    required this.vtigerVersion,
  });

  factory Result.fromJson(Map<String, dynamic> json) {
    return Result(
      sessionName: json['sessionName'],
      userId: json['userId'],
      version: json['version'],
      vtigerVersion: json['vtigerVersion'],
    );
  }
}

File: datacrm_app/android/app/src/main/kotlin/com/example/datacrm_app/models/LoginModel.kt
package com.example.datacrm_app.models

class LoginModel {
    data class ApiResponseGetKey(
        val success: Boolean,
        val result: GetKeyResult
    )

    data class GetKeyResult (
        val token: String,
        val serverTime: Long,
        val expireTime: Long
    )

    data class ApiResponseLogin(
        val success: Boolean,
        val result: LoginResult?
    )

    data class LoginResult(
        val sessionName: String,
        val userId: String,
        val version: String,
        val vtigerVersion: String
    )
}